core/src/main/java/edu/wpi/grip/core/events/ConnectionRemovedEvent.java

```diff
@@ -11,19 +11,19 @@
  * user deleting a connection with the GUI.
  */
 public class ConnectionRemovedEvent implements DirtiesSaveEvent {
-  private final Connection connection;
+  private final Connection<?> connection;
```

Comment:
Because java can't resolve `getSource()` in `ConnectionRemovedEvent.getConnection().getOutputSocket().getSource()` if this is a raw type